src/Adapter/MailTemplate/MailPreviewVariablesBuilder.php

```diff
@@ -119,6 +121,19 @@ public function buildTemplateVariables(LayoutInterface $mailLayout)
         return $templateVars;
     }
 
+    /**
+     * @param $id
+     * @param array $parameters
+     * @param null $domain
+     * @param null $local
+     *
+     * @return string
+     */
+    protected function trans($id, $parameters = [], $domain = null, $local = null)
+    {
+        return $this->context->getTranslator()->trans($id, $parameters, $domain, $local);
```

Comment:
If you need the translator it's better to directly inject `@translator` in the constructor